spec/menu_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "menu"
+
+describe Menu do
+
+  let(:printer_class) { double :printer_class, :format => 0 }
```

Comment:
Think about making the values returned by doubles meaningful, so you can check for the expected value to be returned. e.g. format should probably return a string, with menu items and prices.

Reply:
I'm a bit confused about this, because I thought the point was that it doesn't matter what is being returned at this point, I'm just checking that the method is being called. It's then in the printer class that I test what is being returned. Or have I got the wrong end of the stick?